src/main/java/com/whocraft/whocosmetics/client/models/WeepingAngelWingsModel.java

```diff
@@ -83,41 +84,57 @@ public WeepingAngelWingsModel() {
     }
 
     @Override
-    public void render(LivingEntity entityIn, float limbSwing, float limbSwingAmount, float ageInTicks, float netHeadYaw, float headPitch, float f5) {
+    public void render(LivingEntity entityIn, float limbSwing, float limbSwingAmount, float ageInTicks, float netHeadYaw, float headPitch, float scaleFactor) {
 
-        setRotationAngles(entityIn, limbSwing, limbSwingAmount, ageInTicks, netHeadYaw, headPitch, f5);
-        this.isChild = false;
-        GlStateManager.pushMatrix();
+        setRotationAngles(entityIn, limbSwing, limbSwingAmount, ageInTicks, netHeadYaw, headPitch, scaleFactor);
+//        this.isChild = false; //Don't set this to false, because every tick isChild will be set to false, so it will always render it in full scale
```

Comment:
Fixes model not being scaled on baby zombies. Before, isChild is getting set to false every tick, so the model won't be accessing the GL scaling code, hence it always renders at full scale.